include/small/small.h

```diff
@@ -236,6 +252,16 @@ small_alloc_check(struct small_alloc *alloc)
 	return slab_cache_check(alloc->cache);
 }
 
+/**
+ * Fill `info' with the information about how `size' bytes would be allocated.
```

Comment:
Nit: Fill 'info' with the information about allocation 'ptr' of size 'size'.

Reply:
Done.